pkg/vere/king.c

```diff
@@ -1201,6 +1202,23 @@ u3_king_vere(c3_c* pac_c,  // pace
 
   u3l_log("vere: saved to %s", bin_c);
 
+  #if defined(U3_OS_linux)
+
+    // This runs `sudo setcap 'cap_net_bind_service,cap_setfcap=+ep'` on the new binary
```

Comment:
Does this only work if you `sudo urbit dock $pier` or `sudo $pier/.run next`? And if not, how?

Reply:
The plan is to have the docs say

`sudo setcap 'cap_net_bind_service,cap_setfcap=+ep' <pier>/.run` 

instead of

`sudo setcap 'cap_net_bind_service=+ep' <pier>/.run`

Then you only ever have to do it once.

Reply:
That's a lot of capability ... I wonder if it'd be better just recommend that people run as root if they don't want to deal with capabilities and/or proxy ports 80/443.

Reply:
I think either way is equally annoying. People tend to not like running software as root though